products/iap/api.yaml

```diff
@@ -140,6 +140,8 @@ objects:
     base_url: 'projects/{{project}}/brands'
     self_link: '{{name}}'
     input: true
+    identity:
+      - name
```

Comment:
This is cool. Can this make Brand's `post_create: templates/terraform/post_create/set_computed_name.erb` go away?

Reply:
At least remove the SetId there.

Reply:
It looks like this removed the need for `set_computed_name` for a whole handful of resources. Mind looking into them @emilymye?

Reply:
yeah, I'll probably do it in a separate PR - this will be redundant in most cases but I don't want to accidently remove a setId that is required